docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.Regression
+{
+    public static class FastForest
+    {
+        // This example requires installation of additional NuGet package
+        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
```

Comment:
This can be xml doc.

Reply:
xml looks busy, and it won't render as xml inside the example. This code goes in the example block. 

---
In reply to: [266665386](https://github.com/dotnet/machinelearning/pull/2999#discussion_r266665386) [](ancestors = 266665386)

Reply:
/// xml are not used in .net examples, so we're sticking to // comments inside samples' code.

---
In reply to: [266722954](https://github.com/dotnet/machinelearning/pull/2999#discussion_r266722954) [](ancestors = 266722954,266665386)